nixos/modules/services/misc/gollum.nix

```diff
@@ -86,7 +90,6 @@ in
           groupName = config.users.groups.gollum.name;
         in ''
         # All of this is safe to be run on an existing repo
-        mkdir -p ${cfg.stateDir}
         git init ${cfg.stateDir}
         chmod 755 ${cfg.stateDir}
         chown -R ${userName}:${groupName} ${cfg.stateDir}
```

Comment:
We can drop the `chmod` and `chown` lines here (especially the `chown` wouldn't work anyway without elevated privileges) as they are already covered with the `tmpfiles` rule.

Reply:
Thank you @primeos! I'll mark `gollum` as reviewed :tada: